protocol/source/served/lsp/protocol.d

```diff
@@ -2237,12 +2237,13 @@ struct DidChangeConfigurationClientCapabilities
 /// You might want to use `RootJsonToken` instead of this type if you want to
 /// deserialize your custom config types.
 @serdeFallbackStruct
-@allowedMethods("workspace/didChangeConfiguration")
+@allowedMethods("workspace/didChangeConfiguration", "served/didChangeConfiguration")
```

Comment:
allowedMethods is only for warning for protocols, instead annotate the overload that uses a non-standard method as such. (there is an error message that tells you what UDA to add)

Reply:
I'm not entirely sure I'm following here.

Reply:
remove the extra allowed method, because it's not a protocol standard. (This file should only contain a 1:1 mapping of the protocol standard)

Reply:
Ah ok.